### ISSUE TITLE
TFT max gradient norm

### ISSUE THREAD

**Issue by jfogelberg** [unit8co/darts#815]
In the Temporal Fusion Transformer paper https://arxiv.org/pdf/1912.09363.pdf, one of the main hyperparameters that they tune is the "Max. gradient norm". Is is possible to tune the max gradient norm with the current implementation in Darts?

Thank you,
Johan

**Comment by dennisbader** [comment]
This here might help: https://pytorch-lightning.readthedocs.io/en/stable/advanced/training_tricks.html#gradient-clipping

You can pass trainer params to the TFTModel with `pl_trainer_kwargs` at model creation.

We haven't tested this before, but let us know if it works.
